restler/checkers/payload_body_checker.py

```diff
@@ -1135,31 +1138,59 @@ def _exec_request_with_new_body(
             # render the data
             rendered_data = seq.resolve_dependencies(rendered_data)
 
-            # substitute if there is UUID suffix
-            original_rendered_data = rendered_data
-            uuid4_suffix_dict = self._get_custom_payload_uuid4_suffix()
-            for uuid4_suffix in uuid4_suffix_dict:
-                suffix = uuid4_suffix_dict[uuid4_suffix]
-                len_suffix = len(suffix)
-                # need the query to partition path and body
+            if not self._skip_uuid_substitution:
```

Comment:
>_skip_uuid_substitution [](http://example.com/codeflow?start=24&length=23)

Could you please add a comment referencing that this setting may be removed when issue #429 is fixed?

Reply:
@marina-p  Well, removing this option later won't be backwards compatible... Is this wise?

(The option is harmless and is off by default.) 

OK?

Reply:
I just added a note with issue #429 so we remember this point.